src/nwb_datajoint/common/common_curation.py

```diff
@@ -151,7 +151,20 @@ def _get_curated_sorting_name(self, key):
 
     @staticmethod
     def _sorting_after_reject(sorting, quality_metrics, reject_params):
+       
+        # Reject units based off of noise_overlap
+        # A dict where each key is a unit_id and its value is a noise overlap score
+        nn_noise_overlaps = quality_metrics['nn_noise_overlap']
         
+        # Not sure what this name ('nn_noise_overlap_threshold') should be
+        nn_noise_overlap_threshold = reject_params['nn_noise_overlap_threshold']
+        
+        # Remove units that had too high a noise overlap. Keep the rest
+        nn_noise_overlap_passed_units_ids = [unit_id for unit_id in sorting.get_unit_ids() if nn_noise_overlaps[unit_id] < nn_noise_overlap_threshold]
```

Comment:
can you apply thresholds for both noise overlap and isi violation at once? that way you need to call select_units just once